src/LiveDevelopment/MultiBrowserImpl/protocol/LiveDevProtocol.js

```diff
@@ -147,31 +147,335 @@ define(function (require, exports, module) {
         }
     }
 
-    function _tagSelectedInLivePreview(tagId, nodeName, contentEditable, allSelectors) {
+    function _findWordInEditor(editor, word, nodeName, tagId, wordContext, wordOccurrenceIndex) {
```

Comment:
Can you give a high level overview of how this works in plain text? It will help me understand the changes. Specifically:
1. How we identify the text in the remote live preview.
2. How it communicates this position and finds the correct text position in the editor.